head	1.4;
access;
symbols
	rpe_20161101:1.1.1.1 rpe:1.1.1;
locks; strict;
comment	@# @;


1.4
date	2017.01.06.10.55.20;	author shadchin;	state Exp;
branches;
next	1.3;
commitid	9xnHP7IHlkW5SIzC;

1.3
date	2017.01.06.10.33.37;	author sthen;	state Exp;
branches;
next	1.2;
commitid	aox7ZyN6w8W5y28U;

1.2
date	2017.01.03.19.17.24;	author landry;	state Exp;
branches;
next	1.1;
commitid	AUU81IxhkQGenx5a;

1.1
date	2016.11.01.13.20.10;	author rpe;	state Exp;
branches
	1.1.1.1;
next	;
commitid	ZXduyTrQaRkCOBgm;

1.1.1.1
date	2016.11.01.13.20.10;	author rpe;	state Exp;
branches;
next	;
commitid	ZXduyTrQaRkCOBgm;


desc
@@


1.4
log
@Add test for py-mindb

ok sthen@@
@
text
@# $OpenBSD: Makefile,v 1.3 2017/01/06 10:33:37 sthen Exp $

COMMENT =		simple SQLite3 store for Python objects

MODPY_EGG_VERSION =	2.0.1
DISTNAME =		minidb-${MODPY_EGG_VERSION}
PKGNAME =		py-minidb-${MODPY_EGG_VERSION}
REVISION =		1

CATEGORIES =		databases

HOMEPAGE =		http://thp.io/2010/minidb/

# BSD
PERMIT_PACKAGE_CDROM =	Yes

MASTER_SITES =		http://thp.io/2010/minidb/

MODULES =		lang/python

TEST_DEPENDS =		devel/py-nose,python3

MODPY_VERSION = 	${MODPY_DEFAULT_VERSION_3}

do-test:
	cd ${WRKSRC} && ${LOCALBASE}/bin/nosetests-3

.include <bsd.port.mk>
@


1.3
log
@Workaround https://bugs.python.org/issue28518 which is breaking urlwatch
following the switch to python 3.6, problem reported by semarie
@
text
@d1 1
a1 1
# $OpenBSD: Makefile,v 1.2 2017/01/03 19:17:24 landry Exp $
d21 2
d24 3
@


1.2
log
@bump ports depending on python3 in databases
@
text
@d1 1
a1 1
# $OpenBSD: Makefile,v 1.1.1.1 2016/11/01 13:20:10 rpe Exp $
d8 1
a8 1
REVISION =		0
@


1.1
log
@Initial revision
@
text
@d1 1
a1 1
# $OpenBSD$
d8 1
@


1.1.1.1
log
@import py-minidb

minidb is a simple SQLite3 store for Python objects.

OK aja@@ sthen@@
@
text
@@
